src/app/shared/pipes/safe-html.pipe.ts

```diff
@@ -10,7 +10,6 @@ export class SafeHtmlPipe implements PipeTransform {
   }
 
   transform(value: any, args?: any): any {
-    value = this.sanitizer.sanitize(SecurityContext.HTML, value);
```

Comment:
Doesn't sound safe to completely remove this security feature. Have you tried any other sanitizers/techniques? Did you face any issues with them?

Reply:
@gpalsingh No bro I didn’t try any other sanitizer techniques.

Reply:
@gpalsingh Added Dompurify package so we can pass allow-lists.